src/smart-components/request/expandable-content.js

```diff
@@ -31,21 +39,9 @@ const ExpandableContent = ({ id, content, state }) => {
         }</Level>
       <Level>
         <LevelItem>
-          <TextContent>
-            <Text className="data-table-detail heading" component={ TextVariants.small }>Portfolio</Text>
-            <Text className="data-table-detail content"
-              component={ TextVariants.h5 }>{ content ? content.portfolio : 'Unknown' }</Text>
-          </TextContent>
-          <TextContent>
-            <Text className="data-table-detail heading" component={ TextVariants.small }>Platform</Text>
-            <Text className="data-table-detail content"
-              component={ TextVariants.h5 }>{ content ? content.platform : 'Unknown' }</Text>
-          </TextContent>
-          <TextContent>
-            <Text className="data-table-detail heading" component={ TextVariants.small }>Comments</Text>
-            <Text className="data-table-detail content"
-              component={ TextVariants.h5 }>{ content ? content.comments : 'Unknown' }</Text>
-          </TextContent>
+          { expandedItem('Portfolio', content ? content.portfolio : 'Unknown') }e
```

Comment:
^^
```jsx
<ExpandedItem ... />
```

And i believe there is a trailing character `e` after closing curly bracket